src/List.test.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { render } from '@testing-library/react';
+
+import List from './List';
+
+const handleClick = jest.fn();
+
+test('Without task', () => {
+  const { container } = render((
+    <List tasks={[]} onClickDelete={handleClick} />
+  ));
```

Comment:
```suggestion
  const { container } = render((
    <List
     tasks={[]}
     onClickDelete={handleClick}
   />
  ));
```

저는 props 가 2개 이상일 때 줄바꿈해서 formatting 해주는데 어떤게 더 나은신 것 같나요?